core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -112,24 +152,28 @@ public DefaultClientRequestContext(
      * @param factory the {@link ClientFactory} which is used to acquire an {@link EventLoop}
      * @param sessionProtocol the {@link SessionProtocol} of the invocation
      * @param request the request associated with this context
+     *
+     * @deprecated Use {@link #DefaultClientRequestContext(ClientFactory, MeterRegistry, Scheme, HttpMethod, String, String, String, ClientOptions, Request)}.
```

Comment:
is it safe to assume internally, armeria makes no calls to this ctor anymore?

Reply:
Yes. I made sure nobody in our code calls this.

Reply:
This ctor = the deprecated ctor, right?

Reply:
yep